### PR TITLE
Update human-biomolecular-data page - Beacons

### PR DESCRIPTION
**Pull request by ainajene**
Following the discussions with BY-COVID WP2 Leads, related to deliverable D2.3, I have added the Beacons with COVID-19 data explained in that deliverable.

### CODE REVIEW

**Comment by bedroesb** [comment]
@ainajene this is great. We were thinking of adding the tools to the general tools and resources list so they are more visible and machine actionable. The process to do this is described here: https://www.infectious-diseases-toolkit.org/contribute/tool-resource-update , but we can for sure also help out with this!

File: _data/tool_and_resource_list.yml [hunk 1]
```diff
@@ -961,3 +961,31 @@
   registry:
     tess: Webin-CLI
   url: https://github.com/enasequence/webin-cli
+- description: CRG Covid Viral Beacon is a tool to find SARS-CoV-2 variability at genomic, amino acid and motif level. 
+  id: viral-beacon
```
Comment (bedroesb) [inline _data/tool_and_resource_list.yml:965]: This one seems to be already available as: "crg-covid-19-viral-beacon"

**Comment by bedroesb** [comment]
@ainajene in my opinion you can add yourself as contributor of the page (+ contributor information in our `_data/CONTRIBUTORS.yml` file)

File: _data/tool_and_resource_list.yml (hunk 1 above)
Comment (bedroesb) [reply _data/tool_and_resource_list.yml:965]: @lauportell thanks btw for adding them!

**Review by bedroesb — APPROVED** [review]
this looks good, but lets maybe wait for @ainajene response regarding attribution as contributor

**Review by bedroesb — APPROVED** [review]
this looks good, but lets maybe wait for @ainajene response regarding attribution as contributor

**Comment by lauportell** [comment]
> @ainajene in my opinion you can add yourself as contributor of the page (+ contributor information in our `_data/CONTRIBUTORS.yml` file)

if I am not mistaken, she is already a contributor in the page :)

**Comment by bedroesb** [comment]
Haha I am blind, than we are good!

**Comment by ainajene** [comment]
Thanks for all your work! :)

I'm checking the page (https://www.infectious-diseases-toolkit.org/data-sources/human-biomolecular-data) and the table is not well formatted. 

Is there a way I can fix it? Thanks

**Comment by bedroesb** [comment]
Good catch @ainajene , it should be fixed now: https://www.infectious-diseases-toolkit.org/data-sources/human-biomolecular-data 


Apparently the Markdown table does not play well together with the Tool snippets. I've changed the table to an HTML table for now.

**Comment by ainajene** [comment]
it looks great now! Thank you both @bedroesb @lauportell

**Pull request merged**